app/public/mockServiceWorker.js

```diff
@@ -0,0 +1,287 @@
+/* eslint-disable */
```

Comment:
This file is generated by MSW to integrate with the browser. Don't worry about reviewing it.